smartgymapi/handlers/sport_schedule.py

```diff
@@ -35,7 +35,7 @@ def get(self):
     @view_config(context=SportScheduleFactory, request_method="POST")
     def post(self):
         if get_sport_schedule_by_name(self.request.json_body['name']):
-            raise HTTPBadRequest(json={'message': 'Sport schedule name already exists'})
+            raise KeyError(json={'message': 'Sport schedule name already exists'})
```

Comment:
nee ik bedoel `self.request.json_body['name']` gaat een `KeyError` geven als je geen name meegeeft in de request body. En aangezien je dit later pas valideerd is dat mogelijk. HTTPBadRequest was wel gewoon prima